app/screens/people-screen/circles/invite-friends-card.tsx

```diff
@@ -0,0 +1,117 @@
+import { Alert, Animated, Easing, Pressable, Share, View } from "react-native"
+
+import { makeStyles, Text } from "@rneui/themed"
+import { GaloyIconButton } from "@app/components/atomic/galoy-icon-button"
+
+import { useRef, useState } from "react"
+
+import { gql } from "@apollo/client"
+import { useInviteQuery } from "@app/graphql/generated"
+
+import crashlytics from "@react-native-firebase/crashlytics"
+import { InviteModal } from "@app/components/invite-modal"
+import { getInviteLink } from "./helpers"
+import { useI18nContext } from "@app/i18n/i18n-react"
+
+gql`
+  query invite {
+    me {
+      username
+    }
+  }
+`
+
+export const InviteFriendsCard = () => {
+  const styles = useStyles()
+  const { LL } = useI18nContext()
+  const [isInviteModalVisible, setIsInviteModalVisible] = useState(false)
+
+  const { data } = useInviteQuery()
+
+  const share = async () => {
+    if (!data?.me?.username) return
+    try {
+      const result = await Share.share({ message: getInviteLink(data.me.username) })
+
+      if (result.action === Share.sharedAction) {
+        if (result.activityType) {
+          // shared with activity type of result.activityType
+        } else {
+          // shared
+        }
+      } else if (result.action === Share.dismissedAction) {
+        // dismissed
+      }
+    } catch (err) {
+      if (err instanceof Error) {
+        crashlytics().recordError(err)
+        Alert.alert(err.message)
+      }
+    }
+  }
+
+  const openInviteModal = () => setIsInviteModalVisible(true)
+
+  const scaleAnim = useRef(new Animated.Value(1)).current
+
+  const breatheIn = () => {
```

Comment:
variable name feels awkward to me

Reply:
The animation is a breathe animation. So Breathe in/out.